src/patternfly/demos/Card/card-content-5.hbs

```diff
@@ -0,0 +1,26 @@
+{{#> card-header}}
+  {{#> l-flex flex--modifier="pf-m-align-right"}}
+    {{#> l-flex flex--modifier="pf-m-grow"}}
+      {{#> brand brand--attribute='src="/assets/images/pf-logo-small.svg" alt="Logo"'}} 
```

Comment:
We probably shouldn't use the brand component for this. The brand was specifically meant for the header, not just a generic image element, though I don't think we've said specifically it can't be used elsewhere. I think this would just be better as an `<img>` or inline `<svg>`. Also I'm assuming a card like this, for amq, apache spark, etc, the patternfly logo would likely be replaced with the individual product logos.